### DISCUSSION TITLE
Can I use multi GPU?

### DISCUSSION THREAD

**Discussion by crummenauerca** [Purfview/whisper-standalone-win#283] [Q&A] [answered]
Can I use multi GPU to transcribe a single audio file? Something like this: `CUDA_VISIBLE_DEVICES=0,1,2,3 ./whisper-faster audio.mp3`

**Comment by Purfview** [comment, marked as answer]
> Can I use multi GPU to transcribe a single audio file?

Single you can't.
But you can try running 4 files or different folders at once like that:

`./whisper-faster audio_1.mp3 --device cuda:0`
`./whisper-faster audio_2.mp3 --device cuda:1`
`./whisper-faster audio_3.mp3 --device cuda:2`
`./whisper-faster audio_4.mp3 --device cuda:3`

**Reply by crummenauerca** [reply]
Understood. I already use multi GPUs with more than one audio file simultaneously, but it would be great to use multi GPUs to transcribe just one file as well. Can this be implemented in the future?

**Reply by Purfview** [reply]
> Can this be implemented

No.
